Examples/Shared/SplashViewController.swift

```diff
@@ -0,0 +1,125 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import StreamChat
+import UIKit
+
+final class SplashViewController: UIViewController {
+    private let userInfo: UserInfo
+    private let token: Token
+    private let completionHandler: () -> Void
+    
+    private lazy var activityIndicatorView: UIActivityIndicatorView = .init(style: .large)
+    private lazy var cancellationLabel: UILabel = {
+        let label = UILabel()
+        label.text = "No connected use found.\nTap the button below to try connect the user with id \(userInfo.id)."
```

Comment:
"use" -> "user"

Reply:
Good catch. Updated and i'll open it for review once the checks are finished. Thanks :)